src/FSharpSource.targets

```diff
@@ -683,6 +719,44 @@
   </Choose>
   <Import Condition="'$(ProjectLanguage)' == 'FSharp'" Project="$(FSharpTargetsPath)"/>
 
+  <!-- Hook up DNX to enable solution refresh of packages -->
+  <PropertyGroup Condition="'$(TargetFramework)'=='coreclr'">
+    
+    <NuGetPackagesPath Condition="'$(NuGetPackagesPath)' == ''">$(FSharpSourcesRoot)\..\packages</NuGetPackagesPath>
+
+    <!-- fix this to be not hacky -->
```

Comment:
Just chatted with @latkin offline, it looks like we can combine FX_NO_THREADPOOL and FX_NO_THREAD.  It looks like they may mean basically the same thing.  Which will reduce the confusion.